PowerEditor/src/tools/NppModernShell/README.md

```diff
@@ -0,0 +1,61 @@
+# Windows 11 Modern UI Context Menu integration
+
+The purpose of this project is to allow Notepad++ to integrate into the new Windows 11 right-click context menu.
+Doing this requires two new things.
+
+* A dll library with some COM objects that the shell can communicate with.
+* A Sparse Package containing the metadata for the COM server.
+
+To build this, the following steps needs to be taken:
+
+1. Build a Release dll file (NppModernShell.dll)
+2. Generate a Sparse Package (NppModernShell.msix)
+3. Sign both of these with signtool.exe
+4. Make sure they are included in the installer, so they are deployed next to the notepad++.exe program.
+5. The installer should, upon installation, install the package.
+6. The installer should, upon uninstallation, uninstall the package.
+
+## Build a Release dll file (NppModernShell.dll)
+Just open the NppModernShell.sln Visual Studio solution, select Release as the build type, and do a Rebuild of the solution.
```

Comment:
I would add this line:
(Require: [Windows 11 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/)

Reply:
Done, will be pushing shortly